extensions/pkg/util/secret/manager/manager_test.go

```diff
@@ -173,7 +173,7 @@ var _ = Describe("SecretsManager Extension Utils", func() {
 			})
 
 			JustBeforeEach(func() {
-				By("creating old CA secrets")
+				By("Create old CA secrets")
```

Comment:
There are still files remaining, where By("") has been used in which string's first letter is not capitalized.
You can find such occurrence using this `By\("[a-z]`

Reply:
Thanks!